gui/src/components/onboarding/pages/ConnectTracker.tsx

```diff
@@ -44,90 +47,41 @@ export function ConnectTrackersPage() {
   const { applyProgress, state, skipSetup } = useOnboarding();
   const navigate = useNavigate();
   const { sendRPCPacket, useRPCPacket } = useWebsocketAPI();
-  const [isSerialOpen, setSerialOpen] = useState(false);
-  const [connectionStatus, setConnectionStatus] =
-    useState<ConnectionStatus>('START-CONNECTING');
+  const [provisioningStatus, setProvisioningStatus] =
+    useState<WifiProvisioningStatus>(WifiProvisioningStatus.NONE);
 
   applyProgress(0.4);
 
   const connectedTrackers = useConnectedTrackers();
 
-  const openSerial = () => {
-    const req = new OpenSerialRequestT();
-    req.auto = true;
-
-    sendRPCPacket(RpcMessage.OpenSerialRequest, req);
-  };
-
   useEffect(() => {
     if (!state.wifi) {
       navigate('/onboarding/wifi-creds');
     }
 
-    openSerial();
+    const req = new StartWifiProvisioningRequestT();
+    req.ssid = state.wifi?.ssid as string;
+    req.password = state.wifi?.password as string;
+
+    sendRPCPacket(RpcMessage.StartWifiProvisioningRequest, req);
     return () => {
-      sendRPCPacket(RpcMessage.CloseSerialRequest, new CloseSerialRequestT());
+      sendRPCPacket(
+        RpcMessage.StopWifiProvisioningRquest,
```

Comment:
`StopWifiProvisioningRquest` should be `StopWifiProvisioningRequest`, unless this is intentional, then this will need to be fixed in the SolarXR-Protocol.

Reply:
@loucass003 boop (this is the only thing preventing this PR from being merged rn as far as I’m aware)